packages/frontend/src/layout/AboutPage/AboutPage.jsx

```diff
@@ -16,24 +16,24 @@ export default function AboutPage() {
     <div className="about-page">
       <section className="about-dark-header">
         <div className="about-header-full">
-          <h3>Introducing scout</h3>
+          <h3>Introducing Scout</h3>
```

Comment:
Can we change this to "Welcome to Scout" or something else? Scout has been introduced a long time ago so this feels outdated